original/en.txt

```diff
@@ -501,17 +494,17 @@ It is not the only app we can use to communicate safely, but it is one of the ea
 
 It's much more secure when the person who we're contacting is using Signal too. It uses our internet connection when we contact another Signal user and normal SMS or minutes when we contact someone who is not using Signal.
 
-Remember! You don't have to get everyone you know to use Signal. Just tell your closest friends and the people you contact most often to install it too and then more and more of our friends will start using it.
+Remember! You don't have to get everyone you know to use Signal. Just tell your closest friends and the people you contact most often to install it too and then more and more of your friends will start using it.
 
 This booklet was made possible thanks to:
 
-• individual donors on GlobalGiving.com – thank you all for your contribution!
-• the Adessium Foundation and Open Society Foundations
+• Individual donors on GlobalGiving.com – thank you all for your contribution!
+• The Adessium Foundation and Open Society Foundations
 
 This document is distributed under a Creative Commons 2.0 Licence (CC BY 2.0)
 http://creativecommons.org/licenses/by/2.0/
 
 » QUEST 6: I SHOULD CHOOSE AS MY PASSWORD …
-[R] …a random combination of letters and special characters.
+[R] …a random combination of letters, numbers and special characters.
```

Comment:
Is 'digits' better than 'numbers'?

Reply:
Good question. I used "numbers" because that's how Keepass calls it.